### ISSUE TITLE
Returning submissionID from `avworkflow_run`?

### ISSUE THREAD

**Issue by amstilp** [Bioconductor/AnVIL#102]
I've been using the Bioconductor `AnVIL` package to programmatically submit workflows, and it's great!

One potential improvement would be to have `avworkflow_run()` also return the submissionId of the job. Right now it returns the config that the workflow was submitted with. However, I can easily get that from `avworkflow_configuration_inputs`, but it's not so straightforward to get the submissionId. (The user has to call `avworkflow_jobs()` and assume that the most recent one is the submission, which is usually but not guaranteed to be true in a shared workspace.)

It does look like the [Rawls API for createSubmission](https://rawls.dsde-prod.broadinstitute.org/#/submissions/createSubmission) returns the submissionId so maybe it's not too difficult to change? I'm also open to creating a pull request to make this change, if welcomed.

**Comment by LiNk-NY** [comment]
Hi Adrienne, @amstilp 

Thanks for creating an issue. Sorry for the delay.

The proposed solution would be to add something like `LastSubmissionId` to the returned `config` and a message such as: 

```r
Workflow submitted; Setting config$LastSubmissionId: d0069ded-9de3-43c3-8556-ceea39badb13
  See avworkflow_jobs() for more details.
```

How does that sound?

**Comment by LiNk-NY** [comment]
Feel free to test with this branch: https://github.com/Bioconductor/AnVIL/tree/workflow_run_submissionId

**Comment by amstilp** [comment]
That sounds great, thanks for implementing!

**Comment by LiNk-NY** [comment]
resolved in version [1.17.2](https://github.com/Bioconductor/AnVIL/commit/2cc3787e507cf40d689385c7a29ae26e284927b7)
